pegasus/sites.v3/hourofcode.com/i18n/en.yml

```
@@ -2534,10 +2534,10 @@
   tutorial_grokimage_longdescription: "Use the programming language Python to make speedy changes to images. Make fun image editing programs and make your own image filter!"
   tutorial_icomputelearn_longdescription: "When you learn to code with iCompute you program your own fantastic interactive stories, games, and animations. Following our creative, fun, activities means that you will develop computational thinking as well as coding skills!"
   tutorial_rpstress_longdescription: "Use Scratch to create a stress ball that you can squash!"
-  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause."
+  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause. #codeisyourvoice"
```

Are we intentionally spelling "favourite" the British way?

---

Adding missing spaces
```suggestion
  tutorial_bsdcause_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. The Support My Cause focuses on designing and coding a landing page that will collect data from supporters of your favourite cause. #codeisyourvoice"
```